ffs/create.go

```diff
@@ -72,6 +72,16 @@ func CreateNestedDir(dirPath string) string {
 	return fullPath
 }
 
+// CreateFilePathInHome creates a filepath and returns the path.
+func CreateFilePathInHome(dirPath string) string {
```

Comment:
I don't see a diff between this function and `CreateNestedDir` - can we just use `CreateNestedDir` instead? https://github.com/runfinch/common-tests/blob/f1e5583d81d5bb9fd550a826963dc320faee7266/ffs/create.go#L64-L73

Reply:
Updated `CreateFilePathInHome` to only construct filepath and not mkdir.